build.properties

```diff
@@ -76,6 +76,18 @@ behat.options.strict = false
 # shows exception backtraces, 3 shows debugging information.
 behat.options.verbosity = 2
 
+# The location where to save the screenshots. If empty, the screenshot will not
+# be created.
+# Example:
+# behat.screenshots_dir = ${tmp.dir}/behat
+behat.screenshots_dir =
```

Comment:
I wouldn't duplicate this line, just leave it commented out. This is the typical pattern in config files, see for example `php.ini`.